ZAPD/ZRoom/ZRoom.cpp

```diff
@@ -643,21 +640,21 @@ Declaration::Declaration(DeclarationAlignment nAlignment, DeclarationPadding nPa
 	arrayItemCnt = nArrayItemCnt;
 }
 
-Declaration::Declaration(std::string nIncludePath, size_t nSize, string nVarType, string nVarName)
+Declaration::Declaration(std::string nIncludePath, size_t nSize, std::string nVarType, std::string nVarName)
 	: Declaration(DeclarationAlignment::None, DeclarationPadding::None, nSize, "")
 {
 	includePath = nIncludePath;
 	varType = nVarType;
 	varName = nVarName;
 }
 
-CommandSet::CommandSet(int32_t nAddress)
+CommandSet::CommandSet(uint32_t nAddress)
 {
 	address = nAddress;
-	commandCount = 9999999;
+	commandCount = UINT32_MAX;
```

Comment:
what is commandCount here?

Reply:
To provide an upper limit to the amount of commands a scene could have. It is set to 9999999 for alternate headers.
Thinking about it, it may be fine.